latest/ug/cluster-management/cost-monitoring-kubecost-view-dashboard.adoc

```diff
@@ -0,0 +1,51 @@
+include::../attributes.txt[]
+
+[.topic]
+[#cost-monitoring-kubecost-dashboard]
+= Access Kubecost Dashboard
+
+[#kubecost-prereqs-dashboard]
+== Prerequisites
+
+. Make sure the kubecost related Pods' state are "Running".
+
+[source,bash,subs="verbatim,attributes"]
+----
+kubectl get pods --namespace kubecost
+----
+
+An example output is as follows.
+
+[source,bash,subs="verbatim,attributes"]
+----
+NAME                                          READY   STATUS    RESTARTS   AGE
+kubecost-cost-analyzer-b9788c99f-5vj5b        2/2     Running   0          3h27m
+kubecost-kube-state-metrics-99bb8c55b-bn2br   1/1     Running   0          3h27m
+kubecost-prometheus-server-7d9967bfc8-9c8p7   2/2     Running   0          3h27m
+----
+
+[#kubecost-dashboard]
+== Access Kubecost Dashboard
+
+. On your device, enable port-forwarding to expose the Kubecost dashboard.
++
+[source,bash,subs="verbatim,attributes"]
+----
+kubectl port-forward deployment/kubecost-cost-analyzer 9090 --namespace kubecost
+----
++
+Alternatively, you can use the <<aws-load-balancer-controller,{aws} Load Balancer Controller>> to expose Kubecost and use Amazon Cognito for authentication, authorization, and user management. For more information, see link:containers/how-to-use-application-load-balancer-and-amazon-cognito-to-authenticate-users-for-your-kubernetes-web-apps[How to use Application Load Balancer and Amazon Cognito to authenticate users for your Kubernetes web apps,type="blog"].
+. On the same device that you completed the previous step on, open a web browser and enter the following address.
++
+[source,bash,subs="verbatim,attributes"]
+----
+http://localhost:9090
+----
++
+You see the Kubecost Overview page in your browser. It might take 5–10 minutes (or more) for Kubecost to gather metrics, depends on your cluster size. You can see your Amazon EKS spend, including cumulative cluster costs, associated Kubernetes asset costs, and monthly aggregated spend.
```

Comment:
Original wording was `You see the Kubecost Overview page in your browser. It might take 5–10 minutes for Kubecost to gather metrics. You can see your Amazon EKS spend, including cumulative cluster costs, associated Kubernetes asset costs, and monthly aggregated spend.`

But it could be vary if cluster size is relatively large, minor tweak to add `... (or more)` and `, depends on your cluster size` after page moved.